src/platforms/javascript/common/configuration/integrations/index.mdx

```diff
@@ -9,6 +9,22 @@ redirect_from:
 
 Sentry provides a number of integrations which are designed to add automatic instrumentation to your application. Some of these integrations are enabled by default, while others have to be explicitly enabled.
 
+<PlatformSection supported={["javascript.nextjs"]}>
+
+<Alert title="Integration Usage In Next.js">
+
+Next.js can operate within three runtimes: the Node.js runtime, the browser runtime, or the Edge runtime. However, it's important to note that not all integrations are compatible with all of these runtimes.
+
+Depending on whether an integration enhances the functionality of a particular runtime, such as the BrowserTracing integration for the browser runtime or the RequestData integration for the Node.js runtime, you can only include these integrations in their respective configuration files:
+
+- For the browser runtime, add integrations to `sentry.client.config.ts`.
+- For Node.js, add integrations to `sentry.server.config.ts`.
+- For the Edge runtime, add integrations to `sentry.edge.config.ts`.
+
+</Alert>
+
+</PlatformSection>
```

Comment:
I think this would be better in regular text vs. a note

```suggestion
<PlatformSection supported={["javascript.nextjs"]}>

## Integration Usage in Next.js

Next.js can operate within three runtimes: the Node.js runtime, the browser runtime, or the Edge runtime. However, it's important to note that not all integrations are compatible with all of these runtimes.

Depending on whether an integration enhances the functionality of a particular runtime, such as the BrowserTracing integration for the browser runtime or the RequestData integration for the Node.js runtime, you can only include these integrations in their respective configuration files:

- For the browser runtime, add integrations to `sentry.client.config.ts`.
- For Node.js, add integrations to `sentry.server.config.ts`.
- For the Edge runtime, add integrations to `sentry.edge.config.ts`.

</PlatformSection>
```

Reply:
> I think this would be even better if we added an easy way for users to see which runtime an integration is for at the top of each integration page.

I fully agree with this. However, this will turn into an `N times M` problem for which we don't have the time to maintain at the moment.